themes/ddbasic/sass/components/footer.scss

```diff
@@ -0,0 +1,130 @@
+@import '../base.scss';
```

Comment:
Describe component contents/purpose, why are these styles grouped? What is the common denominator?